src/docker/Dockerfile

```diff
@@ -52,43 +52,31 @@ RUN apt-get update && apt-get install -y \
     automake autoconf
```

Comment:
>and check locally if anything is missing

@mkordas check how? I only know to build the image and run it, how is that relevant to the project, and how it's built by rultor behing the scenes - I have no clue about that